src/Sulu/Bundle/ResourceBundle/Resources/public/js/components/filters/components/form/main.js

```diff
@@ -136,7 +136,7 @@ define(['config'], function(Config) {
                     options: {
                         el: $element,
                         fieldsUrl: typeConfig.fields,
-                        operatorsUrl: constants.operatorsUrl,
+                        operatorsUrl: constants.operatorsUrl + '?locale=' + AppConfig.getUser().locale,
```

Comment:
Do we still have to use the user locale here? That's because the filters themselves are not localized, right?